Src/Sarif.PatternMatcher/SearchDefinition.cs

```diff
@@ -19,6 +21,8 @@ public class SearchDefinition
 
         public ResultKind Kind { get; set; }
 
+        public RuleEnabledState RuleEnabledState { get; set; }
```

Comment:
>RuleEnabledState [](http://example.com/codeflow?start=32&length=16)

This all looks right but where's the release note? What confidence do we have you've made a good change? Ad hoc testing from the command-line? Can we create a simple unit test somewhere appropriate?

Reply:
I created some tests that verify functionality, and intend to use Yong's shell PR (https://dev.azure.com/mseng/1ES/_git/Spmi/pullrequest/698351) which has many rules disabled in this fashion to ensure that everything still works.

Reply:
Roger that. Let's add a release note, clean up indentation in that one location and get this in. :)